config.json

```diff
@@ -1686,7 +1686,7 @@
       "uuid": "01211dc2-c8e1-4621-99a2-6aa66080839a",
       "slug": "multiple-clause-functions",
       "name": "Multiple Clause Functions",
-      "blurb": "TODO: add blurb for multiple-clause-functions concept"
+      "blurb": "In Elixir, a single function can have multiple clauses. This is achieved by pattern matching the function's arguments and by using guards."
```

Comment:
Unfortunately pattern-matching comes *after* multiple clause functions, but I think it's important to mention it in the blurb.